### PR TITLE
add country codes to alternative names

### PR DESCRIPTION
**Pull request by ajaits**
Added lower case country codes to alternative names for places

### CODE REVIEW

**Review by pradh — APPROVED** [review]
Thanks!

File: internal/store/files/WorldGeosForPlaceRecognitionAlternateNames.csv [hunk 1]
```diff
@@ -1,4 +1,159 @@
...
+country/CZE,"cze,cz"
+country/DEU,deu
+country/DJI,dji
+country/DMA,dma
+country/DNK,dnk
+country/DZA,dza
+country/EGY,egy
+country/ERI,eri
+country/ESH,esh
+country/ESP,es
```
[the remark this answers is not in the document]
Comment (ajaits) [reply internal/store/files/WorldGeosForPlaceRecognitionAlternateNames.csv:38]: dropped es

File: internal/store/files/WorldGeosForPlaceRecognitionAlternateNames.csv [hunk 2]
```diff
@@ -1,4 +1,159 @@
...
+country/PSE,pse
+country/PYF,pyf
+country/RWA,rwa
+country/SAU,"sau,sa"
+country/SCG,scg
+country/SDN,sdn
+country/SGP,sgp
+country/SGS,sgs
+country/SHN,shn
+country/SLV,sv
```
[the remark this answers is not in the document]
Comment (ajaits) [reply internal/store/files/WorldGeosForPlaceRecognitionAlternateNames.csv:125]: dropped sv

File: internal/store/files/WorldGeosForPlaceRecognitionAlternateNames.csv [hunk 3]
```diff
@@ -1,4 +1,159 @@
...
+country/USA,usa
+country/UZB,uzb
+country/VCT,vct
+country/VGB,vgb
+country/VNM,vnm
+country/VUT,vut
+country/WLF,wlf
+country/WSM,wsm
+country/XKS,xks
+country/ZAF,"zaf,za"
```
[the remark this answers is not in the document]
Comment (ajaits) [reply internal/store/files/WorldGeosForPlaceRecognitionAlternateNames.csv:157]: dropped za, retained zaf

**Pull request merged**